src/widgets/StepperInputs/TemperatureEditor.js

```diff
@@ -22,13 +24,20 @@ export const TemperatureEditor = ({
   textInputStyle,
   suffixTextStyle,
   stepAmount,
-  maximum,
-  minimum,
+  threshold,
+  above,
+  belowMinimum,
+  aboveMaximum,
 }) => {
+  const minimum = above ? threshold : belowMinimum;
+  const maximum = above ? aboveMaximum : threshold;
+
   const formatted = keepInRange(value, minimum, maximum).toFixed(2);
 
-  const onIncrement = () => onChange(value + stepAmount);
-  const onDecrement = () => onChange(value - stepAmount);
+  const onIncrement = () =>
+    onChange(keepInRange(temperature(value + stepAmount).temperature(), minimum, maximum));
+  const onDecrement = () =>
+    onChange(keepInRange(temperature(value - stepAmount).temperature(), minimum, maximum));
```

Comment:
continuation of the logic required from the selectors to ensure each configs temperatures don't cross over